lib/src/main/java/fi/epicbot/toster/model/Action.kt

```diff
@@ -55,7 +55,7 @@ internal fun Action.title(): String {
         Action.ClearAppData -> "Clear app data"
         Action.ClearText -> TODO()
         is Action.Click -> "Click to ($x;$y)"
-        Action.ClickBack -> TODO()
+        Action.ClickBack -> "Click back"
```

Comment:
Do we have a plan to emulate other clicks? Maybe pass parameter here as some click?

Reply:
yeah, click on item with some id, if this id exists.
but not right now